pages/community/[slug].tsx

```diff
@@ -37,15 +41,50 @@ const postBody = ({ content, post }) => {
   return replacedContent;
 };
 
-export default function Post({ post, posts, preview }) {
+export default function Post({ post, posts, reviewAuthorDetails, preview }) {
   const router = useRouter();
   const morePosts = posts?.edges;
+  const [avatarImgSrc, setAvatarImgSrc] = useState("");
+  const time = 10 + calculateReadingTime(post?.content);
+  const [blogWriterDescription, setBlogWriterDescription] = useState("");
+  const [reviewAuthorName, setreviewAuthorName] = useState("");
+  const [reviewAuthorImageUrl, setreviewAuthorImageUrl] = useState("");
+  const [reviewAuthorDescription, setreviewAuthorDescription] = useState("");
+  const [postBodyReviewerAuthor, setpostBodyReviewerAuthor] = useState(0);
+  
+  useEffect(() => {
+    if (reviewAuthorDetails && reviewAuthorDetails.length > 0) {
+      const authorIndex = post.ppmaAuthorName === "Neha" ? 1 : 0;
+      const authorNode = reviewAuthorDetails[authorIndex]?.edges[0]?.node;
+      if (authorNode) {
+        setpostBodyReviewerAuthor(authorIndex);
+        setreviewAuthorName(authorNode.name);
+        setreviewAuthorImageUrl(authorNode.avatar.url);
+        setreviewAuthorDescription(authorNode.description);
+      }
+    }
+  }, [post, reviewAuthorDetails]);
+  const blogwriter = [
+    {
+      name: post?.ppmaAuthorName || "Unknown Author",
+      ImageUrl: avatarImgSrc,
```

Comment:
please ensure that fallback is added for all the values

Reply:
I have added the fallback values in both the slugs for the community and technology.